src/nbl/asset/interchange/CGLTFLoader.cpp

```diff
@@ -40,17 +40,13 @@ namespace nbl
 
 		namespace SAttributes
 		{
-			_NBL_STATIC_INLINE_CONSTEXPR uint8_t POSITION_ATTRIBUTE_ID = 0;
-			_NBL_STATIC_INLINE_CONSTEXPR uint8_t NORMAL_ATTRIBUTE_ID = 1;
-			_NBL_STATIC_INLINE_CONSTEXPR uint8_t UV_ATTRIBUTE_BEGINING_ID = 2;			//!< those attributes are indexed
-			_NBL_STATIC_INLINE_CONSTEXPR uint8_t COLOR_ATTRIBUTE_BEGINING_ID = 5;		//!< those attributes are indexed
-			_NBL_STATIC_INLINE_CONSTEXPR uint8_t JOINTS_ATTRIBUTE_BEGINING_ID = 8;		//!< those attributes are indexed
-			_NBL_STATIC_INLINE_CONSTEXPR uint8_t WEIGHTS_ATTRIBUTE_BEGINING_ID = 12;	//!< those attributes are indexed
-
-			_NBL_STATIC_INLINE_CONSTEXPR uint8_t MAX_UV_ATTRIBUTES = 3;				
-			_NBL_STATIC_INLINE_CONSTEXPR uint8_t MAX_COLOR_ATTRIBUTES = 3;
-			_NBL_STATIC_INLINE_CONSTEXPR uint8_t MAX_JOINTS_ATTRIBUTES = 4;
-			_NBL_STATIC_INLINE_CONSTEXPR uint8_t MAX_WEIGHTS_ATTRIBUTES = 4;
+			_NBL_STATIC_INLINE_CONSTEXPR uint8_t POSITION_ATTRIBUTE_LAYOUT_ID = 0;
+			_NBL_STATIC_INLINE_CONSTEXPR uint8_t NORMAL_ATTRIBUTE_LAYOUT_ID = 1;
+			//_NBL_STATIC_INLINE_CONSTEXPR uint8_t TANGENT_ATTRIBUTE_LAYOUT_ID = 2; // TODO - missing tangent!
+			_NBL_STATIC_INLINE_CONSTEXPR uint8_t UV_ATTRIBUTE_BEGINING_LAYOUT_ID = 2;			//!< those attributes are indexed
```

Comment:
keep the same ordering of Position, Normal, UV and Color as other formats.

Reply:
outdated